examples/pipelines/datacomp/components/load_from_hf_hub/fondant_component.yaml

```diff
@@ -27,6 +27,8 @@ produces:
   
   image_text:
     fields:
+      uid: 
```

Comment:
Shouldn't this be used as the index? Is this missing functionality in the current `read_from_hub` component?

Reply:
Currently the `load_from_hf_hub` creates a monotonically increasing index: https://github.com/ml6team/fondant/blob/92942afa0d43e150ce77b15f7cc4039d75f222f1/components/load_from_hf_hub/src/main.py#L62-L66.

@PhilippeMoussalli 's PR will remove that and set it in Fondant: https://github.com/ml6team/fondant/pull/345

Reply:
Yes, but the `uid` is a field from the initial huggingface dataset we're loading? Then I would assume that ideally, we use that column as the index in Fondant?

Reply:
Ok but there's no way to specify that currently, right? Would we need to add an `index_column` argument to the `load_from_hf_hub` component?

Reply:
Yes, that's what I was steering towards. No need to add it in this PR, but important to know that that would be useful.

@PhilippeMoussalli related to my comment on #345